packages/perseus/src/widgets/deprecated-standin/deprecated-standin.tsx

```diff
@@ -0,0 +1,46 @@
+import Banner from "@khanacademy/wonder-blocks-banner";
+import React from "react";
+
+import {PerseusI18nContext} from "../../components/i18n-context";
+import noopValidator from "../__shared__/noop-validator";
+
+import type {Widget, WidgetExports} from "../../types";
+
+// The props are type `any` on purpose so that this can receive props
+// from any deprecated widget
+type Props = any;
+
+class DeprecatedStandin extends React.Component<Props> implements Widget {
```

Comment:
It was bugging me that `DeprecatedStandin` was declared in `index.tsx` rather than `deprecated-standin.tsx`. It makes it hard to find and none of our other widgets do that.